rollup-plugins/production-pruning.js

```diff
@@ -63,24 +63,28 @@ const formatMessage = (node) => {
 
 const createErrorDocumentation = (node) => {
   const errorMessage = formatMessage(node);
-  const errorIndex = errors.push(errorMessage);
+  let errorIndex = errors.indexOf(errorMessage);
```

Comment:
I would avoid let whenever we can. What if instead of defining error index here we define it as a const on line 71 and just do this check on line 68 without assigning it?

I.e. if (errors.includes(errorMessage)) {}

Reply:
Makes a lot of sense. Done.